docs/examples/bike-sharing-forecasting.ipynb

```diff
@@ -460,9 +460,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The score we now have is much more realistic, as it is comparable with [related data science competitions](https://maxhalford.github.io/blog/a-short-introduction-and-conclusion-to-the-openbikes-2016-challenge/). Moreover, we can see that the model gets better with time, which feels better than the previous situations. The point is that `online_qa_score` method can be used to simulate a production scenario, and is thus extremely valuable.\n",
+    "The score we now have is much more realistic, as it is comparable with [related data science competitions](https://maxhalford.github.io/blog/a-short-introduction-and-conclusion-to-the-openbikes-2016-challenge/). Moreover, we can see that the model gets better with time, which feels better than the previous situations. The point is that `progressive_val_score` method can be used to simulate a production scenario, and is thus extremely valuable.\n",
```

Comment:
This link seems to be broken

Reply:
Yes, it's actually https://maxhalford.github.io/blog/openbikes-challenge/